### PR TITLE
feat(CORSO-31): view all faq

### PR DESCRIPTION
**Pull request by DenisaNicoletaH**
JIRA:  
https://champlainsaintlambert.atlassian.net/browse/CORSO-31
https://champlainsaintlambert.atlassian.net/browse/CORSO-38

## Context:
This feature consists of displaying all of the FAQs as well as the preferred FAQs on the main page.
The preferred FAQs were supposed to be on a different pull request but accidently did two in one

## Changes

- Added FAQ files
- Implemented FAQ service
- Implemented Preferred FAQs Service
- Achieved 90% Jacoco Coverage
- Selenium Tests Achieved
- Organized files in the backend

## Before and After UI (Required for UI-impacting PRs)
FAQ Page: 
image

FAQ Main Page:
image

### CODE REVIEW

File: corso_website_fe/src/Pages/AllFAQs/ViewAllFaq.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                </div>
+                <div className={"text-center text-md-end me-0 me-md-5"}>
+                    <a href={"/#faq"}>
+                        Back To Main Page
+                    </a>
+                </div>
+                <div className={"tiles-container"}>
+                    {faqs.map((value) => (
+                        <FaqTileFaqPage
+                            //@ts-ignore
```
Comment (Mila5847) [inline corso_website_fe/src/Pages/AllFAQs/ViewAllFaq.tsx:50]: can you please explain what is the //ts-ignore doing?

**Review by Mila5847 — APPROVED** [review]
looks nice, just like the Figma! The BE is tested and you also have E2E testing, nice!

File: corso_website_fe/src/Pages/AllFAQs/ViewAllFaq.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            })
+            .catch(e => {
+                console.log(e)
+            })
+    }
+
+    useEffect(() => {
+        getFaqs()
+
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (KarinaSofia) [inline corso_website_fe/src/Pages/AllFAQs/ViewAllFaq.tsx:31]: could you explain what this is doing?

**Review by KarinaSofia — APPROVED** [review]
great work implementing faq. all the testing looks good, and the front end looks incredibly accurate to the planned fe design great work! Also good work fixing up little things with the project file too!

File: corso_website_fe/src/Pages/AllFAQs/ViewAllFaq.tsx (hunk 2 above)
Comment (DenisaNicoletaH) [reply corso_website_fe/src/Pages/AllFAQs/ViewAllFaq.tsx:31]: This stops the CI/CD from blocking.

File: corso_website_fe/src/Pages/AllFAQs/ViewAllFaq.tsx (hunk 1 above)
Comment (DenisaNicoletaH) [reply corso_website_fe/src/Pages/AllFAQs/ViewAllFaq.tsx:50]: It ignores specific errors done within a block of code or a line of code

**Pull request merged**